### ISSUE TITLE
something wrong with x_capture

### ISSUE THREAD

**Issue by Kyupii** [Kyupii/CheToolbox#28]
```py
eq_est = separations.eq_curve_estim(eq_points)
feed_line = separations.mccabe_thiel_feedline(1,.60)
eq_feedpoint = (.60,eq_est.eval(.60))
separations.mccabe_thiel_otherlines(feed_line,eq_feedpoint,x_D,x_B,1.3)
separations.mccabe_thiel_full_est(eq_est,feed_line,x_F,x_D,x_B,1.3,PLOTTING_ENABLED=True)
```
returns error:
```py
{
	"name": "ValueError",
	"message": "setting an array element with a sequence. The requested array has an inhomogeneous shape after 2 dimensions. The detected shape was (2, 2) + inhomogeneous part.",
	"stack": "---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In[3], line 5
      3 eq_feedpoint = (.60,eq_est.eval(.60))
      4 separations.mccabe_thiel_otherlines(feed_line,eq_feedpoint,x_D,x_B,1.3)
----> 5 separations.mccabe_thiel_full_est(eq_est,feed_line,x_F,x_D,x_B,1.3,PLOTTING_ENABLED=True)

File c:\\PY\\yt_downloader\\CheToolbox\\chetoolbox\\separations.py:398, in mccabe_thiel_full_est(eq_curve, feedline, xf, xd, xb, Rmin_mult, tol, PLOTTING_ENABLED)
    395   linestograph.append(common.point_separsort((x, y), (eq_curve.inv(y), y)))
    396   return y
--> 398 ideal_stages = common.curve_bouncer(eq_curve, y_operlines, xd, xb, x_graphcapture, y_graphcapture)
    400 if PLOTTING_ENABLED:
    401   fig, ax = plt.subplots(); ax.set_title(\"McCabe Thiel Diagram\")

File c:\\PY\\yt_downloader\\CheToolbox\\chetoolbox\\common.py:363, in curve_bouncer(upper, lower, x_start, x_stop, x_transform, y_transform)
    361 xprev = x
    362 if x_transform != None:
--> 363   x = x_transform(x)
    364 y = lower.eval(x)
    365 if y_transform != None:

File c:\\PY\\yt_downloader\\CheToolbox\\chetoolbox\\separations.py:391, in mccabe_thiel_full_est.<locals>.x_graphcapture(x)
    389 def x_graphcapture(x):
    390   y = eq_curve.eval(x)
--> 391   linestograph.append(common.point_separsort((x, y), (x, y_operlines.eval(x))))
    392   return x

File c:\\PY\\yt_downloader\\CheToolbox\\chetoolbox\\common.py:291, in point_separsort(*points)
    287 def point_separsort(*points: list | tuple | npt.NDArray) -> tuple[npt.NDArray, npt.NDArray]:
    288   '''
    289   Takes an arbitrary set of points and sorts them by x-value. Returns them as two np.arrays, one of x-values and the other of y-values.
    290   '''
--> 291   points = np.atleast_1d(points).reshape(-1, 2)
    292   points = points[points[:, 0].argsort()]
    293   return points[:, 0], points[:, 1]

File c:\\Users\\quang\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\
umpy\\core\\shape_base.py:65, in atleast_1d(*arys)
     63 res = []
     64 for ary in arys:
---> 65     ary = asanyarray(ary)
     66     if ary.ndim == 0:
     67         result = ary.reshape(1)

ValueError: setting an array element with a sequence. The requested array has an inhomogeneous shape after 2 dimensions. The detected shape was (2, 2) + inhomogeneous part."
}
```

**Comment by Kyupii** [comment]
Thought I could figure this out quickly but it's taking longer...

**Comment by ethanmolnar06** [comment]
fixed with commit 19e3b24bd3cd2964fb3370e965b94c1512752ce6
